staking/app/scripts/snapshot.ts

```diff
@@ -105,48 +105,57 @@ async function main() {
   const stakeAccounts = await getAllStakeAccounts(RPC_URL);
   const profileAccounts = await getAllProfileAccounts(RPC_URL);
 
-  const stakers: { owner: PublicKey; stakedAmount: BN }[] = stakeAccounts.map(
```

Comment:
The diffs here are very bad on github... but in reality I'm just adding timeOfFirstStake at two or three places